.github/ISSUE_TEMPLATE/epic.md

```diff
@@ -13,6 +13,17 @@ about: Template for our high-level story published in the external roadmap
 
 ---
 
+<!--
+Please assign appropriate labels:
+* product (for product area, choose between `product/KaaS`, `product/managed-services`, `product/empowerment`)
```

Comment:
do we wanna turn `product/KaaS` into `product/kaas`? that seems easier on my brain

Reply:
yes, Marian asked the same